packages/components/checkbox/src/Checkbox.tsx

```diff
@@ -9,8 +9,8 @@ export type CheckboxProps = InputProps
 export const Checkbox = React.forwardRef<
   React.ElementRef<typeof CheckboxPrimitive.Root>,
   CheckboxProps
->(({ children, ...props }, ref) => (
-  <Label data-spark-component="checkbox" disabled={props.disabled}>
+>(({ children, className, ...props }, ref) => (
+  <Label data-spark-component="checkbox" className={className} disabled={props.disabled}>
```

Comment:
@Powerplex Is this ok to pass `className` in label, not input?

Reply:
That's a good question. I think so yes. But it means we can add styles to the whole block not the checkbox itself (but maybe this is good).

For compound components we won't have this question because every subComponent will be able to receive its own `className` :) Let's keep it like you did.

Reply:
If in the future if someone need to pass different styles to the label wrapping the component VS **only on the input**, we might need to introduce a dedicated prop for it.